### DISCUSSION TITLE
Index Bundle – large amount of columns affecting performance?

### DISCUSSION THREAD

**Discussion by NiklasBr** [coreshop/CoreShop#2502] [Q&A]
Have not yet installed this bundle, but are currently looking into using it to help simplify our product listing pages. 

In our installation PIM editors can choose pretty much any field on a Product DataObject to be used for display and sorting in the  customer-facing front end listings, including any from a few hundred classification store keys.

My understanding is that the bundle will create one db column per field configured, so, are there any practical limitations where you think that there are too many columns? E g where this bundle may not provide a performance benefit (or we may need to look into a way to limit the number of columns).

**Comment by dpfaffenbauer** [comment]
@NiklasBr Well, if you have hundreds of columns, you should look into ElasticSearch. There is a open PR for Elastic Integration with Index. We usually don't let the Customer change the index, we do it for them.

**Reply by NiklasBr** [reply]
I could give it a try. Are you able to mirror the PR or branch into `coreshop/index-bundle` so I can use `composer require coreshop/index-bundle`?

**Reply by dpfaffenbauer** [reply]
It is mirrored automatically.
